ui/components/multichain/app-header/index.scss

```diff
@@ -73,8 +73,8 @@
   }
 }
 
-.multichain-app-header-shadow {
-  box-shadow: var(--shadow-size-md) var(--color-shadow-default);
+.multichain-app-header-border {
+  border-bottom: 1px solid var(--color-border-muted);
```

Comment:
Nice change. Tested how this looks when content is scrolled, seems good.
(I did like the subtle shadow though rip)

<details><summary>before/after screenshots</summary>


| Before (subtle shadow) | After (hard border) |
|--------|--------|
| ![Cell](https://github.com/user-attachments/assets/8b2e9065-f4e3-4f34-a692-313a7848ffa0) | ![Cell](https://github.com/user-attachments/assets/5f65e03f-ee88-4c16-b958-ec38890c5f08) | 

</details>